HHS/Head-Start-TTADP
# [TTAHUB-1229] Fixes for hooks where unlocking caused errors

## Description of change
Fixed two errors in the activity report hooks when unlocking a report. One was a syntax error with a reduce, the other had the transaction being committed while still executing a hook.

## How to test
Unlock a report. Observe there are no errors. Confirm that the additional hooks are being fired as expected.

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1229

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
